README.md

```diff
@@ -12,5 +12,5 @@ like GitLab CI and Travis CI are available in
 The cache generated by coala are placed on `/cache`. You can preserve it by
 creating a volume for it, like this:
 
-`docker run -v=$(pwd):/app -v=/tmp/coala:/cache --workdir=/app coala/base 
+`docker run -v=$(pwd):/app -v=/tmp/coala:/cache --workdir=/app coala/base
```

Comment:
This is an unrelated change to this commit. Make a separate commit to remove the trailing whitespace please?